cmd/cueckoo/cmd/runtrybot.go

```diff
@@ -68,16 +68,20 @@ func runtrybotDef(cmd *Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	r := newCLTrigger(cmd, cfg, func(payload clTriggerPayload) error {
-		p, err := buildTryBotPayload(payload)
+	r := newCLTrigger(cmd, cfg, func(payload repositoryDispatchPayload) error {
+		trybotPayload := payload
+		trybotPayload.Type = string(eventTypeTrybot)
```

Comment:
make the Type field use the same named type for consistency?

Reply:
I delibeately did not do that in order that the `repositoryDispatchPayload` more closely matches the CUE type which is the source of truth. There's more consolidation/refactoring we could do there, but this is sufficient for now IMO.